pkg/storage/system_intake.go

```diff
@@ -432,7 +429,6 @@ func (s *Store) FetchSystemIntakesByEuaID(ctx context.Context, euaID string) (mo
 	intakes := []models.SystemIntake{}
 	const whereClause = `
 		WHERE system_intakes.eua_user_id=$1
-			AND system_intakes.status != 'WITHDRAWN'
```

Comment:
double check

Reply:
removed a hard-coded conditional in SQL - is this ok?

Reply:
perhaps we should just replace this check with the following:
```suggestion
			AND system_intakes.archived_at IS NOT NULL
```

That way when someone archives a request, it doesn't show up again in the list of intakes for that user.

Reply:
I'm very smart. Not only is that the exact _wrong_ filter, the _correct_ filter is already placed in this query, just one line below.

I'll keep this around for posterity, but I think the final answer is that you're good to just remove this status check as you did!